src/utils/Bootstrap.cpp

```diff
@@ -99,34 +103,89 @@ int Bootstrap::init(int argc, char** argv)
     llvm::InitializeNativeTarget();
```

Comment:
If we have include the `llvm` namespace why prefix this functions with the `llvm` scope? Is it needed?